set.mm

```diff
@@ -26078,15 +26078,92 @@ the definition of class equality ( ~ df-cleq ).  Its forward implication
+       it still accepts ` x e. A ` being a primitive syntax term, not assuming
+       any specific semantics (like elementhood in some form).
+
+       Use it in contexts where you want to avoid ~ df-clab , or you need
+       ~ df-cleq anyway.  See ~ cbvexeqsetv-ab , not using ~ df-cleq or
+       ~ ax-ext or ~ ax-9 .  (Contributed by Wolf Lammen, 6-Aug-2025.)
+       (Proof modification is discouraged.) $)
+    cbvexeqsetv $p |- ( E. x x = A <-> E. y y = A ) $=
+      ( vz cv wceq wex cbvexeqsetvlem bitr4i ) AECFAGDECFDGBECFBGADCHBDCHI $.
+    $( $j usage 'cbvexeqsetv' avoids 'ax-8' 'ax-10' 'ax-11' 'ax-12' 'ax-13'
+       'df-clab'; $)
+  $}
+
+  ${
+    $d x A $.  $d x z $.
+    $( Lemma for ~ cbvexeqsetv-ab .  Proven without ~ ax-9 and ~ ax-ext .
+       (Contributed by BJ, 24-Apr-2024.)
+       (Proof modification is discouraged.) $)
+    cbvexeqsetvablem $p |- ( E. x x = A <-> A e. { z | T. } ) $=
+      ( cv wceq wex wtru cab wcel wa vextru biantru exbii dfclel bitr4i ) ADZCE
+      ZAFQPGBHZIZJZAFCRIQTASQBAKLMACRNO $.
+
+    $d y z A $.
+    $( Use this theorem in contexts where ~ df-cleq or ~ ax-ext is not
+       referenced elsewhere in your proof.  The suffix "-ab" indicates, it is
+       proven from a specific implementation (axiom or class builder
+       ~ df-clab ) of ` x e. A ` .  This is different from ~ cbvexeqsetv
+       allowing ` x e. A ` being a primitive syntax term without any specific
+       semantics.
+
+       This theorem would be the justification for the definition of the unary
+       predicate "E!" by ` |- ( ` E! ` A <-> E. x x = A ) ` which could be
+       interpreted as " ` A ` exists" (as a set) or " ` A ` denotes" (in the
+       sense of free logic).
+
+       A shorter proof using ~ bitri (to add an intermediate proposition
+       ` E. z z = A ` with a fresh ` z ` ), ~ cbvexvw , and ~ eqeq1 , requires
+       the core axioms and { ~ ax-9 , ~ ax-ext , ~ df-cleq } whereas this proof
+       requires the core axioms and { ~ ax-8 , ~ df-clab , ~ df-clel }.
+
+       That every class is equal to a class abstraction is proved by ~ abid1 ,
+       which requires { ~ ax-8 , ~ ax-9 , ~ ax-ext , ~ df-clab , ~ df-cleq ,
+       ~ df-clel }.
+
+       Note that there is no disjoint variable condition on ` x , y ` but the
+       theorem does not depend on ~ ax-13 .  Actually, the proof depends only
+       on the logical axioms ~ ax-1 through ~ ax-7 and ~ sp .
+
+       The symbol "E!" was chosen to be reminiscent of the analogous predicate
+       in (inclusive or non-inclusive) free logic, which deals with the
+       possibility of nonexistent objects.  This analogy should not be taken
+       too far, since here there are no equality axioms for classes: these are
+       derived from ~ ax-ext and ~ df-cleq (e.g., ~ eqid and ~ eqeq1 ).  In
+       particular, one cannot even prove ` |- E. x x = A => |- A = A ` without
+       ~ ax-ext and ~ df-cleq .
```

Comment:
I suppose the correct symbol here is `E.` ?

Reply:
I copied it right from bj-denotes, in the hope he knows what he talks about.  I think on some deeper philosophical insight gives a meaning to E! A, somehoe expressing it is not a proper class, but a set.

Reply:
Sorry for the delay, I'm going to have a look at what should be in the main part and what I'll keep in the mathbox.

The symbol is `E!` and indeed I know what I'm talking about ! But probably the sentences about free logic are best kept in my mathbox. I'm going to make a proposal.